src/fadec/src/EngineControl.h

```diff
@@ -60,8 +60,8 @@ class EngineControl {
   int engineImbalanced;
   double paramImbalance;
 
-  const double LBS_TO_KGS = 0.453592;
-  const double KGS_TO_LBS = 2.20462;
+  const double LBS_TO_KGS = 0.4535934000001;
+  const double KGS_TO_LBS = 2.2046176156881;
```

Comment:
Can you elaborate on the source of these. Are they MSFS constants, or standard conversion factors (IIRC there is a discrepancy between those)?

Reply:
I've set the conversion being used by the ATSU/ AOC code. Since I was not able to find the actual conversion factors in the code, I used SimvarWatcher to check PAYLOAD STATION simvars both in Lbs and Kgs.

Reply:
We should be using conversations consistent with what MSFS uses internally, regardless of how "correct" they are.

Reply:
It would be good to see a comment in the code noting where the numbers come from and why.